README.md

```diff
@@ -99,7 +114,7 @@ To deploy this app follow the steps below:
     Note that this will likely spawn multiple processes as "nobody"
     depending on the number of server ports specified.
     
- #. The ``bin/django.wsgi`` script should make it 
+ 1. The ```bin/django.wsgi``` script should make it 
  	easy to deploy this using mod_wsgi.  You will need to add a line of the form:
 
         WSGIScriptAlias / "/var/www/online_test/bin/django.wsgi"
```

Comment:
Same here, no bin directory or django.wsgi.

It should probably be something like this

WSGIScriptAlias / "/var/www/online_test/online_test/wsgi.py